src/plugins/OauthPlugin/OauthPlugin.js

```diff
@@ -6,28 +8,43 @@ import { DialogPlugin } from '@/plugins/DialogPlugin';
 import { MessengerGroupPlugin } from '@/plugins/MessengerGroupPlugin';
 import OauthApi from '@/plugins/OauthPlugin/OauthPublicApi';
 import { DIRECTION, PLUGIN_METHODS, PLUGIN_NAMES } from '@/constants';
-import oauthHandlers from './oauthHandlers';
+import oauthHandlers from '@/plugins/OauthPlugin/oauthHandlers';
 import FrameStrategy from '@/plugins/OauthPlugin/FrameStrategy';
 
 const documentsCheckReg = /\/documents$/gi;
 
 export default class OauthPlugin extends PluginBase {
+  /**
+   * @returns {string}
+   */
   static get pluginName() {
     return PLUGIN_NAMES.OAUTH;
   }
 
+  /**
+   * @returns {OauthHandlers}
+   */
   static get handlers() {
     return oauthHandlers;
   }
 
+  /**
+   * @returns {[typeof DialogPlugin, typeof MessengerGroupPlugin]}
```

Comment:
is it fine that returns `typeof`? we not create instances here, we just return classes as is

Reply:
@dudiq without `typeof` TS will request `new` before class names. So in this case it's correct.